scripts/generate_contributors_list.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Is this a safe assumption? I've had some issues with this in the past, so I now use `/usr/bin/env bash`

Reply:
I think this is fine as is. I'm guessing this is a direct copy of the swift-nio contributors code. It'll only ever be run by the maintainers of the project.